src/@components/@common/hooks/useOutClickCloser.ts

```diff
@@ -2,15 +2,22 @@ import { useEffect, useRef } from "react";
 
 type HandleOutClickCloser = () => void;
 
-export default function useOutClickCloser(handleOutClickCloser: HandleOutClickCloser) {
+interface useOutClickCloserProps {
+  preventEventPropagation: boolean;
+}
+
+export default function useOutClickCloser(handleOutClickCloser: HandleOutClickCloser, props: useOutClickCloserProps) {
+  const { preventEventPropagation } = props;
```

Comment:
props 라기보다는 parameter 개념에 좀 더 가깝지 않을까요??!

```suggestion
export default function useOutClickCloser(handleOutClickCloser: HandleOutClickCloser, isPreventEventPropagation : boolean) {
```

Reply:
듣고 보니, 파라미터처럼 주고 받긴하네요?!?!?😲

Reply:
`preventEventPropagation` 앞에 is를 붙이는 기준은 무엇일까요?

Reply:
@NYeonK 
boolean 값이기 떄문에요!

아래 클린코드 내용 한 번 읽어보시는 거 추천합니닷
https://github.com/Yooii-Studios/Clean-Code/blob/master/Chapter%2002%20-%20%EC%9D%98%EB%AF%B8%20%EC%9E%88%EB%8A%94%20%EC%9D%B4%EB%A6%84.md#%EB%A9%94%EC%84%9C%EB%93%9C-%EC%9D%B4%EB%A6%84

Reply:
이해됐습니다:) 좋은 자료 감사해용🤩